src/test/scala/ru/smslv/akka/dns/AsyncDnsResolverSpec.scala

```diff
@@ -99,4 +101,43 @@ class AsyncDnsResolverSpec extends AkkaSpec(
       answer.ipv6 should be(empty)
     }
   }
+
+  "Resolver functions" should {
+    "parse a resolv.conf file successfully" in {
+      val resolvConf = Files.createTempFile("resolve", "conf").toFile
+      resolvConf.deleteOnExit()
+      Files.write(
+        resolvConf.toPath,
+        """# This file is automatically generated.
+          |#
+          |search lan
+          |nameserver 10.0.1.1
+          |nameserver 2001:44b8:3127:3500::1
+          |
+        """.stripMargin.getBytes
+      )
+      AsyncDnsResolver.parseSystemNameServers(resolvConf) should be(Some(List(
+        AsyncDnsResolver.parseNameserverAddress("10.0.1.1"),
+        AsyncDnsResolver.parseNameserverAddress("2001:44b8:3127:3500::1")
+      )))
+    }
+
+    "return an empty list when a resolv.conf file has no nameservers" in {
+      val resolvConf = Files.createTempFile("resolve", "conf").toFile
+      resolvConf.deleteOnExit()
+      Files.write(
+        resolvConf.toPath,
+        """# This file is automatically generated.
+          |#
+          |search lan
+        """.stripMargin.getBytes
+      )
+      AsyncDnsResolver.parseSystemNameServers(resolvConf) should be(Some(List.empty))
+    }
+
+    "return nothing when there is no resolv.conf" in {
+      val resolvConf = new File("something-somewhere")
+      AsyncDnsResolver.parseSystemNameServers(resolvConf) should be(None)
+    }
+  }
```

Comment:
Correct. We don't actually validate resolv.conf files... I don't really want to introduce a parser here. We just ensure that we extract it correctly. However we now have a log message if no resolvers can be found. I shall additionally include a message indicating the resolvers that can be found:

```
[INFO] [10/20/2016 19:43:50.703] [AsyncDnsResolverSpec-akka.actor.default-dispatcher-5] [akka://AsyncDnsResolverSpec/system/IO-DNS/async-dns/$a] Using the following DNS nameservers: /8.8.8.8:53, /8.8.4.4:53
```